pkg/ambex/main.go

```diff
@@ -192,6 +196,13 @@ func parseArgs(ctx context.Context, rawArgs ...string) (*Args, error) {
 		dlog.Errorf(ctx, "Invalid AMBASSADOR_AMBEX_SNAPSHOT_COUNT: %s, using %d", numsnapStr, args.numsnaps)
 	}
 
+	// edsByPass will bypass using EDS and will insert the endpoints into the custer data manually
+	// This is a stop gap solution to resolve 503s on certification rotation
+	edsByPass := os.Getenv("AMBASSADOR_EDS_BY_PASS")
```

Comment:
We do have a helper method `envBool`. It's behavior basically ignores the value of the os.Environment variable and is more of a if the environment variable exists with any value then it is true. I'm not sure I agree with that behavior (need to think about it) but it might be good to be consistent?

take a look and let me know your thoughts...

Reply:
I am not a fan of ignoring the value of the env var. I say we leave it as is but I added in `strings.ToLower` so that if a user puts in True or true (or any variant of true) it will be valid.